psi4/src/psi4/scfgrad/response.cc

```diff
@@ -420,8 +420,30 @@ std::shared_ptr<Matrix> RSCFDeriv::hessian_response()
             psio_->write(PSIF_HESS,"Vpi^A",(char*)Vpip[0], static_cast<size_t> (nmo) * nocc * sizeof(double),next_Vpi,&next_Vpi);
         }
     }
+#if 0
```

Comment:
debug? will the ACS compile?

Reply:
excellent catch!  this is something i stuck in there while debugging (I usually grep for ACS when done).  I completely forgot in this case, but that code should not be in there: those terms are already handled [here](https://github.com/psi4/psi4/blob/42ecb0859b36112f17f021981f4e7d8e18ad86ef/psi4/src/psi4/scfgrad/scf_grad.cc#L1020-L1034).  I'll nuke the offending lines in my cleanup pass